### DISCUSSION TITLE
KeyboardAdapter not working with Arduino IDE

### DISCUSSION THREAD

**Discussion by pilot-mefiu** [forntoh/LcdMenu#265] [General]
Hi All,
any idea what can be wrong with the KeyboardAdapter not working with ArduinoIDE and Arduino Nano?
I can successfully run other examples, including rotary encoder controlling the menu, but not KeyboardAdapter.
It works perfectly fine with PlatformIO.

**Comment by B1ackt34** [comment]
I have the same prob ... edited example to use rotary encoder
